indico/cli/i18n.py

```diff
@@ -335,6 +336,55 @@ def remove_empty_pot_files(path: Path, python=False, javascript=False, react=Fal
         _get_messages_react_pot(path).unlink()
 
 
+# Merge after generating all the .pot files
+def merge_pot_files(output_file: Path, *input_files: list[Path]):
+    """Merge multiple POT files into a single POT file."""
+    merged_catalog = Catalog(project='Indico', version=indico.__version__)
+
+    for input_file in input_files:
+        with input_file.open('rb') as f:
+            catalog = read_po(f)
+
+        assert catalog is not None
+
+        for message in catalog:
+            if message.id not in merged_catalog:
```

Comment:
`Catalog` implements `__setitem__` which takes care of duplicates and singular/plural messages ;)
```suggestion
```